js-miniapp-sdk/src/modules/user-info.ts

```diff
@@ -28,9 +28,14 @@ export interface UserInfoProvider {
 
   /**
    * Fetches the access token from host app.
+   * @param audience one of the audiences provided in MiniApp manifest
+   * @param scopes scopes array associated to the audience
    * @returns Access token from native host app.
    */
-  getAccessToken(): Promise<AccessTokenData>;
+  getAccessToken(
+    audience?: string,
+    scopes?: string[]
```

Comment:
I think these properties shouldn't be optional. The Mini App is always required to pass these or else the iOS/Android SDK will return an error.

Reply:
I'm just confused after discussing with @minh-rakuten, I just have to re-focus on this point. For now it's working with the current PR on iOS side, I really have to check with @minh-rakuten as it's the format I understood he also expected 😭

Reply:
I think we can have `getAccessToken(): Promise<AccessTokenData>;` as deprecated function and the new one `getAccessToken(audience: string, scopes: string[]): Promise<AccessTokenData>;`

Reply:
sorry that comment was for the previous feedback

Reply:
I don't know @minh-rakuten. The old method will always fail because now scopes will be mandatory. That's a good notice